charm-slurmd/src/charm.py

```diff
@@ -529,6 +531,13 @@ def singularity_install(self, event):
                 logger.error(f"## Missing singularity resource - {e}")
                 event.fail(message=f'Error installing Singularity: {e.output}')
 
+    def mpi_install(self, event):
+        """Install MPI (mpich)."""
+        self._slurm_manager.mpi.install()
+
+        if self._slurm_manager.mpi.installed():
+            event.set_results({'installation': 'Successfull.'})
```

Comment:
In case the installation fails, how is the user notified of it? How would one see the error message?

Reply:
There is an error message in the log: https://github.com/omnivector-solutions/slurm-ops-manager/blob/master/slurm_ops_manager/mpi.py#L61

Reply:
What do you think about being explicit about the error to the end user when the actions fails?

Something like
```python
event.fail(message=f'Error installing mpich: {error_message}')
```

Reply:
Yes, It's a good thing! But I don't have the `error_message` at this point.

I could just do:

`event.fail(message='Error installing mpich. Check the logs.')`

What do you think?